CMS-Enterprise/easi-app
# [EASI-4467] trb related requests table

# EASI-4467

## Description

- display Related Requests in TRB additional information (same component as System Intake Table)
- added flag for `trbRelatedRequests` and removed `systemIntakeRelatedRequests` flag from codebase (will delete on LD after this ships). will then delete `trbRelatedRequests` entirely after confirmed good to go

## How to test this change

- turn on LD feature flag for `trbRelatedRequests`
- nav to home page as admin
- open a trb request (which has related requests, might have to click through a few to find one)
- click Additional Information
- confirm table appears

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
